src/Css.rei

```diff
@@ -524,6 +524,16 @@ let alignItems:
   [ | `stretch | `flexStart | `center | `flexEnd | `baseline] => rule;
 let alignSelf:
   [ | `stretch | `flexStart | `center | `flexEnd | `baseline | `auto] => rule;
+let justifySelf:
+  [
+    | `flexStart
+    | `center
+    | `flexEnd
+    | `spaceBetween
+    | `spaceAround
```

Comment:
I don't find the values spaceBetween and spaceAround in the https://developer.mozilla.org/en-US/docs/Web/CSS/justify-self doc